wiki/People/Global_Moderation_Team/de.md

```diff
@@ -11,65 +11,67 @@ tags:
 
-Generell wird das Team Menschen über mehrere Wochen oder Monate hinweg beobachten, um sicher zu gehen, dass diese gut ins Team passen. Bitte sei geduldig und schieße nicht über das Ziel hinaus. Du darfst gerne jemanden des GMTs über Regeln, das Melden von Spielern und andere Probleme fragen.
+Generell wird das Team Kandidaten über mehrere Wochen oder Monate hinweg beobachten, um sicherzugehen, dass diese gut ins Team passen. Bitte sei geduldig und schieße nicht über das Ziel hinaus. Du darfst den globalen Moderatoren gerne Fragen zu Regeln, dem Melden von Spielern und anderen Problemen stellen.
 
 ### Einige Dinge, die du nicht machen solltest
 
-1. Frage das GMT nicht ständig, ob du als potentieller Kandidat geeignet bist. Das GMT möchte niemanden hinzufügen, der nur als Mittel zum Zweck beiträgt. Ein altruistisches Mindset ist der Schlüssel.
-2. Melde nicht jede Kleinigkeit im Chat. Es ist eine wichtige Fähigkeit, zu wissen, was gemeldet werde sollte und was nicht.
-3. Beantworte nicht Fragen von anderen, ohne dass du weißt, ob deine Antwort korrekt ist oder nicht; insbesondere in den Foren. Vorschläge sind willkommen, aber das Herausgeben von Falschinformationen hilft keinem. Wenn du die Lösung für ein Problem nicht kennst, dann lasse jemand anderes antworten und lerne von ihm.
-4. Mache nicht zu viel. Das Team schaut auf einen konsistenten Beitrag und nicht auf eine überladene Menge. Es ist bekannt, dass das Menschen die Geduld und das Engagement verlieren lässt. Nur keine Hektik!
+1. Frage das GMT nicht ständig, ob du als potenzieller Kandidat geeignet bist. Das GMT möchte niemanden hinzufügen, der nur zum Selbstzweck beiträgt. Deine Herangehensweise sollte nicht eigennützig sein.
+2. Melde nicht jede Kleinigkeit im Chat. Du solltest unterscheiden können, was gemeldet werden sollte und was nicht.
+3. Beantworte keine Fragen, wenn du dir mit der Antwort nicht sicher bist. Das gilt vor allem in den Foren. Vorschläge sind willkommen, aber das Verbreiten von Falschinformationen hilft niemandem. Wenn du die Lösung für ein Problem nicht kennst, dann lasse jemand anderen antworten und lerne von ihm.
+4. Mache nicht zu viel. Das Team sucht nicht nach übermäßig vielen, sondern nach regelmäßigen Beiträgen. Es ist bekannt, dass Kandidaten sonst ihre Geduld und ihr Engagement verlieren. Nur keine Hektik!
 
 ## Vorteile
 
-Globale Moderatoren erhalten die folgenden Vorzüge:
+Globale Moderatoren erhalten die folgenden Vorteile:
 
 - **[osu!supporter](/wiki/osu!supporter)**: Ein osu!supporter-Tag, das für die Dauer der Mitgliedschaft im Team gilt.
-- **Abzeichen für das Engagement**: Ein [Profilabzeichen](/wiki/Community/Profile_badge), das die Anzahl der Jahre anzeigt, die die Person im GMT war (beginnend mit 3).
+- **Abzeichen für das Engagement**: Ein [Profilabzeichen](/wiki/Community/Profile_badge), das die Amtszeit des Nutzers, beginnend mit 3 Jahren, anzeigt.
```

Comment:
```suggestion
- **Abzeichen für das Engagement**: Ein [Profilabzeichen](/wiki/Community/Profile_badge), das die Beteiligung des Nutzers, beginnend mit 3 Jahren, anzeigt.
```
Maybe a bit too formal, though both are okay

Reply:
`Beteiligung` kind of removes the context of time. I'm open to changing away from `Amtszeit`, but maybe include the time aspect?

Reply:
I can't think of a word that does express the time aspect as much as `Amtszeit` does, so let's just keep it for now.